.gitignore

```diff
@@ -133,3 +133,6 @@ experiments/
 
 # don't track replays
 data/AllCompressed/
+
+# storage for b9 scripts
+run_scripts/
```

Comment:
Might be overkill, but this is where I'm storing `run-slippi.sh` on my local machine.